lighthouse-extension/app/src/lighthouse-background.js

```diff
@@ -148,11 +167,22 @@ window.runLighthouseInExtension = function(options, requestedAggregations) {
   // Default to 'info' logging level.
   log.setLevel('info');
   const connection = new ExtensionProtocol();
-  return connection.getCurrentTabURL()
+  return enableOtherChromeExtensions(false)
+    .then(_ => connection.getCurrentTabURL())
     .then(url => window.runLighthouseForConnection(connection, url, options, requestedAggregations))
     .then(results => {
+      // Deliver results even if enabling extensions fails.
+      enableOtherChromeExtensions(true).catch(err => {
+        log.warn('Chrome', `Could not enable some extensions. ${err.message}`);
```

Comment:
should upgrade to `log.error`, but will `enableOtherChromeExtensions` ever actually reject? [The docs](https://developer.chrome.com/extensions/management#method-setEnabled) mention that it *could* fail but don't mention *how* it will fail. Presumably it won't throw but will do the callback with an error or set `lastError` or something? We should hook that up to the `reject` callback